libsolidity/formal/SMTChecker.cpp

```diff
@@ -58,6 +58,19 @@ void SMTChecker::analyze(SourceUnit const& _source)
 		_source.accept(*this);
 }
 
+bool SMTChecker::visit(ContractDefinition const& _contract)
+{
+	for (auto _var : _contract.stateVariables())
```

Comment:
This does not include inherited state variables, so we have to be careful once we add inheritance.